storybook/storybook-react/src/SkipLink/SkipLink.stories.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Grid, Paragraph, Screen, SkipLink } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Navigation/Skip Link',
+  component: SkipLink,
+  args: {
+    children: 'Direct naar inhoud',
+    href: '#',
+  },
+  argTypes: {
+    style: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+  decorators: [
+    (Story) => (
+      <Screen>
+        <Grid>
+          <Grid.Cell span="all">
+            <Story />
+          </Grid.Cell>
+        </Grid>
+      </Screen>
+    ),
+  ],
+} satisfies Meta<typeof SkipLink>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    style: {
```

Comment:
Do we need to explain this in a code comment or when introducing the story?

Reply:
A code comment is a good idea I think. In Storybook the difference is already explained in the Story itself, I think? Or isn't that obvious enough?